### PR TITLE
build(deps): bump @shopify/flash-list and palette-mobile

### PR DESCRIPTION
**Pull request by gkartalis**
This PR resolves [PHIRE-957] 

### Description

build(deps): bump @shopify/flash-list and palette-mobile

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- bump @shopify/flash-list and react-native-collapsible-tab-view - gkartalis

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[PHIRE-957]: https://artsyproduct.atlassian.net/browse/PHIRE-957?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Dev changes (bump @shopify/flash-list and react-native-collapsible-tab-view - gkartalis)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against d7ce1faf77fd0851179a6842192e19cb285d28ca
</p>

File: package.json [hunk 1]
```diff
@@ -157,6 +157,7 @@
     "react-native-blurhash": "1.1.11",
     "react-native-bootsplash": "3.2.0",
     "react-native-code-push": "8.1.0",
+    "react-native-collapsible-tab-view": "7.0.1",
```
Comment (gkartalis) [inline package.json:160]: we prob don't need this after we bump the palette dep, will remove it

**Comment by gkartalis** [comment]
tested out in both android and iOS and works as expected ⭐️

**Review by brainbicycle — APPROVED** [review]
💯 nice one!

**Pull request merged**